.github/workflows/daily_fuzz.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
         run: cargo build --locked
       - name: Fuzz
         run: |
+          # shellcheck disable=SC2046
```

Comment:
It might be ok to quote this:

```sh
"$(shuf ...)"
```

Although I don't really have a linux machine to test. I'd be fine to ignore this.

Reply:
I checked it on my Mac, and the command copied-and-pasted into my shell works as it currently is on `main`, but adding the quoting breaks it. It's possible it might be different on Linux, but it's also just quite nice to use commands in our GitHub workflows that I can test locally

Reply:
I see, let's keep it disabled then. Thanks for testing it.